andela/AndelaEatsUI
# #160237246 View vendors on admin dashboard

#### What does this PR do?
- Adds functionality for viewing vendors on the admin dashboard page
#### Description of Task to be completed?
- Fetch and replace static contents with dynamic contents
#### How should this be manually tested?
- Install this chrome extension [here](https://chrome.google.com/webstore/detail/allow-control-allow-origi/nlfbmbojpeacfghkpbjhddihlkkiljbi?hl=en-US)
- Click on the extension icon and toggle `on` to enable it
- Log in to the app
- Switch to the admin dashboard from the user profile dropdown menu
- Click on vendors on the side navigation menu
- You should see vendors displayed
#### What are the relevant pivotal tracker stories?
- [ #160237246](https://www.pivotaltracker.com/story/show/160237246)
#### Background Context
There are currently some issues concerning cors which necessitates the need for installing the `Allow-Control-Allow_Origin` extension as a temporary fix for now. However the cveat is that the mock data being served from apiary stops working whenever the chrome extension installed is active and will need to be deactivated for the user page to work.
The admin endpoints are currently being bypassed by manually generating an admin token on the frontend which is sent everytime with the request payload for admin priviledged routes pending the resolution of a more scalable implementation.
#### Screenshots

![image](https://user-images.githubusercontent.com/26303683/45173446-22e55880-b200-11e8-9d79-81a194e7cb80.png)
